### PR TITLE
support TYPE_DATETIME to es date (RFC3339 UTC)

### PR DESCRIPTION
**Pull request by henter**

### CODE REVIEW

**Comment by siddontang** [comment]
Thanks @henter 

Does This PR has the same purpose with #135? 

Please add test.

File: river/sync.go [hunk 1]
```diff
@@ -311,6 +311,13 @@ func (r *River) makeReqColumnData(col *schema.TableColumn, value interface{}) in
 		if err == nil && f != nil {
 			return f
 		}
+	case schema.TYPE_DATETIME:
+		switch v := value.(type) {
+		case string:
+			vt, _ := time.Parse("2006-01-02 15:04:05", string(v))
```
Comment (siddontang) [inline river/sync.go]: seem you can use mysql.TimeFormat directly.

File: river/sync.go [hunk 2]
```diff
@@ -311,6 +311,13 @@ func (r *River) makeReqColumnData(col *schema.TableColumn, value interface{}) in
...
+			timezone := time.Now().Local().Format("-0700")
```
Comment (siddontang) [inline river/sync.go]: why -0700 here?

Comment (henter) [reply river/sync.go]: Fixed. Thanks  :-)

**Comment by siddontang** [comment]
hi @henter 

Can you add a test for this PR?

**Comment by henter** [comment]
https://github.com/siddontang/go-mysql-elasticsearch/pull/143/files#diff-f40063141edb3fde05eff252daf208eeR241

@siddontang test updated, not sure if bugs or errors, thanks.

File: elastic/client.go [hunk 3]
```diff
@@ -249,6 +259,45 @@ func (c *Client) CreateMapping(index string, docType string, mapping map[string]
...
+	}
+
+	defer resp.Body.Close()
+
+	data, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+
+	var m map[string]map[string]map[string]map[string]map[string]interface{}
```
Comment (siddontang) [inline elastic/client.go]: horrible type here, seem it is better to define a structure here.

**Comment by henter** [comment]
@siddontang mapping struct updated.  thanks.

**Pull request merged**